.azuredevops/platformPipelines/platform.updateReadMe.yml

```diff
@@ -13,7 +13,7 @@ trigger:
       - 'modules/**/deploy.json'
 
 variables:
-  - template: '../../global.variables.yml'
+  - template: '../../settings.yml'
```

Comment:
Is this used in this pipeline?

Reply:
I haven't recognized any variables in the pipeline that uses variables from the existing 'global.variables.yml'. Don't mind taking it off, it is an outcome from a find and replace, and I would prefer not to modify existing changes outside of the scope of this PR, but thumbs up if that's the preference.

Reply:
I would suggest removing the references in another clean up PR. @MrMCake thoughts?

Reply:
Fine by me